devops/openshift/api-deploy.yml

```diff
@@ -240,6 +237,16 @@ objects:
       minReplicas:  "${{REPLICAS}}"
       maxReplicas: 10
       targetCPUUtilizationPercentage: 80
+  - apiVersion: policy/v1
+    kind: PodDisruptionBudget
+    metadata:
+      name: ${NAME}-pdb
+    spec:
+      selector:
+        matchLabels:
+          app: ${NAME}
+      maxUnavailable: 1
+      disruptionsAllowed: 1
```

Comment:
Do you mind elaborating on how the `disruptionsAllowed` configuration affects the PDB?

Reply:
https://kubernetes.io/docs/tasks/run-application/configure-pdb/

![image](https://github.com/bcgov/SIMS/assets/62901416/23040bf3-488e-467b-854e-6047524aee21)

disruptionsAllowed is a status configuration, I thought i can configure them. Removing it now.

Reply:
I was not sure about what it was after reading some docs that why I asked 😉